docs/src/snap/tutorial/getting-started.md

```diff
@@ -19,6 +19,12 @@ Install the {{product}} snap with:
 sudo snap install k8s --edge --classic
 ```
 
+```{warning}
```

Comment:
I think a warning here is a bit strong. I didn't take into account where the tutorial will fail is directly below the requirements section. I think this would be better suited in the "what you will need" section just above - along the lines of: A system without any previous installations of containerd/docker. Installing either with {{product}} may cause conflicts.  and then link to the LXD page as an alternative.

Reply:
Done